src/main.js

```diff
@@ -72,10 +79,36 @@ class OfferSelector {
       body: JSON.stringify(sourceIds)
     };
 
-    return fetch(this.options.offers, init)
+    const offersUrl = `${this.options.query}/search/offers`;
+    return fetch(offersUrl, init)
       .then(parseResponse)
-      .then(response => parser.parseOffers(response.data, this.options))
-      .then(val => this.displayOffers(val))
+      .then(response => {
+        //If there are offers, parse the offers
+        if (response.data.length !== 0) {
+          return offerParser.parseOffers(response.data, this.options)
+            .then(result => Promise.resolve([result, 'offer']));
+        //If there are no offers look for licensors
+        } else {
+          return licensorParser.parseLicensors(sourceIds, this.options)
+            .then(result => Promise.resolve([result, 'licensor']));
+        }
+      })
+      //If there are no offers or licensors, look for links.
+      .then(response => {
```

Comment:
calling the parameter `response` is a bit misleading, maybe expand it into `([response, type]) => { ... }`